extensions/amp-story/1.0/amp-story-interactive-quiz.js

```diff
@@ -89,19 +89,7 @@ export class AmpStoryInteractiveQuiz extends AmpStoryInteractive {
    * @param {Element} root
    */
   attachContent_(root) {
-    const promptContainer = root.querySelector(
-      '.i-amphtml-story-interactive-quiz-prompt-container'
-    );
-
-    if (!this.element.hasAttribute('prompt-text')) {
-      this.rootEl_.removeChild(promptContainer);
-    } else {
-      const prompt = document.createElement('p');
-
-      prompt.textContent = this.element.getAttribute('prompt-text');
-      prompt.classList.add('i-amphtml-story-interactive-quiz-prompt');
-      promptContainer.appendChild(prompt);
-    }
+    this.attachPrompt_(root);
```

Comment:
Abstracted away the attachPrompt part, that relies on just having the i-amphtml-story-interactive-prompt-container class